ctre/basemotorcontroller.py

```diff
@@ -72,14 +75,14 @@ def getDeviceID(self):
         """
         return self.getDeviceNumber()
 
-    def set(self, mode: ControlMode, demand0: float, demand1: float = 0.0):
+    def set(self, mode: ControlMode, demand0: float, demand1: float = 0.0, demand1Type: DemandType = DemandType.Neutral):
```

Comment:
Now that this takes 4 arguments, our WPI wrapper classes need to acknowledge this. I personally think using `match_arglist` there is mostly a bad idea anyway.